src/components/slidingPanel/js/SlidingPanel.js

```diff
@@ -112,4 +139,22 @@ var fluid_2_0 = fluid_2_0 || {};
         that.events[that.model.isShowing ? "onPanelShow" : "onPanelHide"].fire();
     };
 
+    // panelId is passed in to ensure that it is evaluated before this
+    // function is called.
+    fluid.slidingPanel.setAriaProperties = function (that, panelId) {
+        that.locate("toggleButton").attr({
```

Comment:
@michelled @amb26 
It depends on your point of view, and what you want to accentuate.  For example, if you're thinking of it as a DOM manipulation, then attr() is fine.  All aria-* are attributes (so is 'role').   But, from the point of view of ARIA, you're setting/getting a state or a property.  Yes, ARIA makes a distinction between states and properties [1], where, to a first approximation, states are subject to change, whereas properties are less so.  In that case, ariaProp() and ariaState() might make more sense (and role()?).  Then again, maybe that's overkill.
[1] http://www.w3.org/TR/wai-aria-1.1/#statevsprop